homeassistant/components/lutron_caseta/light.py

```diff
@@ -48,37 +57,164 @@ async def async_setup_entry(
 
 
 class LutronCasetaLight(LutronCasetaDeviceUpdatableEntity, LightEntity):
-    """Representation of a Lutron Light, including dimmable."""
+    """Representation of a Lutron Light, including dimmable, white tune, and spectrum tune."""
 
-    _attr_color_mode = ColorMode.BRIGHTNESS
-    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
     _attr_supported_features = LightEntityFeature.TRANSITION
 
+    SUPPORTED_COLOR_MODE_DICT = {
+        DEVICE_TYPE_SPECTRUM_TUNE: {
+            ColorMode.HS,
+            ColorMode.COLOR_TEMP,
+            ColorMode.WHITE,
+        },
+        DEVICE_TYPE_WHITE_TUNE: {ColorMode.COLOR_TEMP},
+    }
+
+    def get_min_color_temp_kelvin(self, light) -> int:
+        """Return minimum supported color temperature.
+
+        :param light: The light to get the minimum color temperature for.
+        """
+        white_tune_range = light.get("white_tuning_range")
+        # Default to 1.4k if not found
+        if white_tune_range is None or "Min" not in white_tune_range:
+            return 1400
+
+        return white_tune_range.get("Min")
+
+    def get_max_color_temp_kelvin(self, light) -> int:
+        """Return maximum supported color temperature.
+
+        :param light: The light to get the maximum color temperature for.
+        """
+        white_tune_range = light.get("white_tuning_range")
+        # Default to 10k if not found
+        if white_tune_range is None or "Max" not in white_tune_range:
+            return 10000
+
+        return white_tune_range.get("Max")
+
+    def __init__(self, light, data) -> None:
+        """Initialize the light and set the supported color modes.
+
+        :param light: The lutron light device to initialize.
+        :param data: The integration data
+        """
+        super().__init__(light, data)
+
+        self._attr_min_color_temp_kelvin = self.get_min_color_temp_kelvin(light)
+        self._attr_max_color_temp_kelvin = self.get_max_color_temp_kelvin(light)
+
+        light_type = light["type"]
+        if light_type in self.SUPPORTED_COLOR_MODE_DICT:
+            self._attr_supported_color_modes = self.SUPPORTED_COLOR_MODE_DICT[
+                light_type
+            ]
```

Comment:
```suggestion
            self._attr_supported_color_modes = SUPPORTED_COLOR_MODE_DICT.get(light_type, {ColorMode.BRIGHTNESS})
```

linter will complain about this suggestion